pkgs/tools/games/ps3-disc-dumper/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildDotnetModule
+, dotnetCorePackages
+, fetchFromGitHub
+, zlib
+, openssl
+}:
+
+buildDotnetModule rec {
+  pname = "ps3-disc-dumper";
+  version = "3.2.3";
+
+  src = fetchFromGitHub {
+    owner = "13xforever";
+    repo = "ps3-disc-dumper";
+    rev = "v${version}";
+    sha256 = "sha256-m3TS9H6cbEAHn6PvYQDMzdKdnOnDSM4lxCTdHBCXLV4=";
+  };
+
+  selfContainedBuild = true;
+
+  projectFile = [ "UI.Console/UI.Console.csproj" ];
```

This doesn't need to be a list:
```suggestion
  projectFile = "UI.Console/UI.Console.csproj";
```